controller/refugee_controller.py

```diff
@@ -1,42 +1,71 @@
 ###---- Manage Refugee Menu ----
 from datetime import date
-
+from enum import Enum
+from typing import Type
+from models.plan import Plan
 from models.camp import Camp
 from models.refugee import Refugee
+from controller.controller_error import ControllerError
+
+
+def list_medical_condition_types() -> Type[Enum]:
+    """
+    List all options for medical condition type.
+    """
+    return Refugee.MedicalCondition()
 
 
-def create_refugee(firstname: str,
+def create_refugee(user_id: str,
+                   firstname: str,
                    lastname: str,
                    camp: Camp,
                    num_of_family_member: int,
                    starting_date: date,
                    medical_condition_type: Refugee.MedicalCondition) -> Refugee:
     """
-    Yingbo, Michelle
-    A function used by admin and volunteer.
+    Function to create and save refugee data.
     """
-    pass
+    try:
+        for plan in Plan.all():
+            plan_name = str(plan.name)
+            curr_plan = Plan.find(plan_name)
+            if camp in curr_plan.camps:
+                new_refugee = Refugee(firstname = firstname,
+                                      lastname = lastname,
+                                      num_of_family_member = num_of_family_member,
+                                      starting_date = starting_date,
+                                      medical_condition_type = medical_condition_type)
```

Comment:
should also have `camp = camp` when creating new refugee

Reply:
When I put camp = camp in the creating new refugee, there was an error with camp. 
Since each refugee profile will be saved in a camp, I think it might not save the camp name under a camp. Do you think it makes sense to add camp = camp here？